service.subtitles.legendasdivx/addon.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.subtitles.legendasdivx"
+       name="LegendasDivx.com"
+       version="1.0.4"
+       provider-name="HiGhLaNdeR">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="vfs.rar" version="2.3.0"/>	
+  </requires>
+  <extension point="xbmc.subtitle.module"
+             library="service.py" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">LegendasDivx.com - Search and Download Subtitles</summary>
+    <description lang="en_GB">Search and download subtitles from LegendasDivx.com. Please check the add-on configuration before using it! Note: The "Force Full Description" switch will grab all the text in the website including all the non useful information. The "Parent Folder Search and Match" set to "AUTO" will first search by parent folder but if it's not a release then it will search normally, it's recommend to leave it "AUTO". Library is always primary this is only for non Library Movies\TVSHOWS.</description>
+    <description lang="pt_PT">Procura e download de legendas do site LegendasDivx.com. Por favor verifica as configurações do add-on antes de usar! Nota: O campo "Forçar Descrição Completa" vai mostrar todo o texto da descrição da legenda, mesmo o que não interessa. O campo "Procura e sincroniza pela pasta do filme" em modo "AUTO" vai primeiro tentar procurar pelo nome da directoria "PAI" mas se a mesma não for uma release irá procurar pelo nome do ficheiro, recomenda-se que esteja em "AUTO". Se o filme ou série estiver na Biblioteca, o nome do filme\serie é da Biblioteca.</description>
+    <disclaimer lang="en_GB">highlander@teknorage.com</disclaimer>
+    <disclaimer lang="pt_PT">highlander@teknorage.com</disclaimer>
+    <language>pt_PT en_GB</language>
```

Comment:
language should be: "pt en" not "pt_PT en_GB"

Reply:
Ok.